src/reanimated2/platform-specific/checkVersion.ts

```diff
@@ -1,4 +1,9 @@
-import { version as jsVersion } from '../../../package.json';
+/**
+ * We hardcode the version of Reanimated here in order to compare it
+ * with the version used to build the native part of the library in runtime.
+ * Remember to keep this in sync with the version declared in `package.json`
```

Comment:
`Remember to keep this in sync with the version declared in package.json` - This is the tricky part 😁
By my experience this is not a good practices to keep something variable hardcoded in code.

Reply:
I would rather prefer to generate file contains Reaniated version (version.json) during package build

Reply:
Or we can keep it in code but we need to add CI to test it

Reply:
I like the last idea 😎

Reply:
I know, but we've tried a ton of different things, and they just refuse to work :(
We'll put a check in the CI that they're in sync, and it should be fine.